BASE/Test/TestFramework/Shared/LocalInProcHttpServer.cs

```diff
@@ -25,13 +32,25 @@ public LocalInProcHttpServer(string url)
                 .UseUrls(url)
                 .Configure((app) =>
                 {
-                    app.Run(ServerLogic);
+                    app.Run(Server);
                 })
                 .Build();
 
             Task.Run(() => this.host.RunAsync(this.cts.Token));
         }
 
+        private Task Server(HttpContext context)
+        {
+            this.RequestCounter++;
+            
+            if (this.ServerSideAsserts != null)
+            {
+                this.ServerSideAsserts(context);
+            }
```

Comment:
```suggestion
            this.ServerSideAsserts?.Invoke(context);
```

Reply:
i tried that and it gave me an error! let me check again...